docs/posts/pydata-performance/index.qmd

```diff
@@ -0,0 +1,691 @@
+---
+title: "Ibis versus X: Performance across the ecosystem part 1"
+author: "Phillip Cloud"
+date: 2023-11-22
+categories:
+  - blog
+  - case study
+  - ecosystem
+  - performance
+---
+
+**TL; DR**: Ibis has a lot of great backends. They're all
+good at different things. For working with local data, it's hard to beat DuckDB
+on feature set and performance.
+
+Buckle up, it's going to be a long one.
+
+## Motivation
+
+Ibis maintainer [Gil Forsyth](https://github.com/gforsyth) recently wrote
+a [post on our
+blog](https://ibis-project.org/posts/querying-pypi-metadata-compiled-languages/)
+replicating [**another** blog
+post](https://sethmlarson.dev/security-developer-in-residence-weekly-report-18)
+but using Ibis instead of raw SQL.
+
+I thought it would be interesting to see how other tools compare to this setup,
+so I decided I'd try to do the same workflow on the same machine using
+a few tools from across the ecosystem.
+
+I chose two incumbents--[pandas](https://pandas.pydata.org/) and
+[dask](https://www.dask.org/)--to see how they compare to Ibis + DuckDB on this
+workload. In part 2 of this series I will compare two newer engines--Polars and
+DataFusion--to Ibis + DuckDB.
+
+I've worked on both Pandas and Dask in the past but it's been such a long time
```

Comment:
s/Pandas/pandas throughout?

Reply:
I originally changed from pandas to Pandas from a previous review.

Reply:
sorry, that's my bad.  I guess pandas prefers lowercase in prose?  Not a huge deal either way -- I think the most important thing is for it to be consistent (which it is)

Reply:
it is always lowercase -- I was just trying to find where that's documented but it's ridiculous difficult to search their site for (or isn't there anymore)

Reply:
Cody is correct here - I can't find the pandas style guide, but an old colleague of mine has a separate repo defining some common pydata style conventions that also agrees here: https://github.com/MrPowers/pydata-style-guide

Reply:
Adjusted.

Reply:
The only variation is that I capitalize Pandas at the start of a sentence. Pandas can't override the rules of English :)